src/services/item/validation.ts

```diff
@@ -1,12 +1,36 @@
 import { Ajv } from 'ajv';
+import { S } from 'fluent-json-schema';
 
-import { ItemGeolocation } from '@graasp/sdk';
+import { Alignment, DescriptionPlacement, ItemGeolocation, MaxWidth } from '@graasp/sdk';
 
 import { Item } from './entities/Item';
-import { geolocation, settings } from './schema';
+
+const geolocation = S.object().prop('lat', S.number()).prop('lng', S.number());
+
+const settings = S.object()
+  // Setting additional properties to false will only filter out invalid properties.
+  .additionalProperties(false)
+  // lang is deprecated
+  .prop('lang', S.string())
+  .prop('isPinned', S.boolean())
+  .prop('tags', S.array())
+  .prop('showChatbox', S.boolean())
+  .prop('isResizable', S.boolean())
+  .prop('hasThumbnail', S.boolean())
+  .prop('ccLicenseAdaption', S.string())
+  .prop('displayCoEditors', S.boolean())
+  .prop('descriptionPlacement', S.enum(Object.values(DescriptionPlacement)))
+  .prop('isCollapsible', S.boolean())
+  .prop('enableSaveActions', S.boolean())
+  // link settings
+  .prop('showLinkIframe', S.boolean())
+  .prop('showLinkButton', S.boolean())
+  // file settings
+  .prop('maxWidth', S.enum(Object.values(MaxWidth)))
+  .prop('alignment', S.enum(Object.values(Alignment)));
```

Comment:
Is it still needed for these schemas to be in fluent? Can't we move fully to typebox?
Or I think this is already defined in another file?

Reply:
It was defined in another file. I suggest to use TypeCompiler from Typebox which should have the same effect. I think it is better to do in another PR
(Source)[https://github.com/sinclairzx81/typebox?tab=readme-ov-file#typecheck-typecompiler]